[2023-06-24T10:50:08.541+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_airflow_dag_v2.spark_submit_task manual__2023-06-24T10:50:08.091301+00:00 [queued]>
[2023-06-24T10:50:08.548+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_airflow_dag_v2.spark_submit_task manual__2023-06-24T10:50:08.091301+00:00 [queued]>
[2023-06-24T10:50:08.548+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-24T10:50:08.548+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-06-24T10:50:08.548+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-24T10:50:08.556+0000] {taskinstance.py:1303} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2023-06-24 10:50:08.091301+00:00
[2023-06-24T10:50:08.561+0000] {standard_task_runner.py:55} INFO - Started process 1732 to run task
[2023-06-24T10:50:08.564+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'spark_***_dag_v2', 'spark_submit_task', 'manual__2023-06-24T10:50:08.091301+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/spark_jobs_dag2.py', '--cfg-path', '/tmp/tmp7e28fole']
[2023-06-24T10:50:08.564+0000] {standard_task_runner.py:83} INFO - Job 61: Subtask spark_submit_task
[2023-06-24T10:50:08.574+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:250: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-06-24T10:50:08.611+0000] {task_command.py:388} INFO - Running <TaskInstance: spark_airflow_dag_v2.spark_submit_task manual__2023-06-24T10:50:08.091301+00:00 [running]> on host 846f24a1d814
[2023-06-24T10:50:08.661+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=spark_***_dag_v2
AIRFLOW_CTX_TASK_ID=spark_submit_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-24T10:50:08.091301+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-24T10:50:08.091301+00:00
[2023-06-24T10:50:08.668+0000] {base.py:73} INFO - Using connection ID 'spark_connn' for task execution.
[2023-06-24T10:50:08.669+0000] {spark_submit.py:341} INFO - Spark-Submit cmd: spark-submit --master  --name arrow-spark /opt/***/dags/spark_etl_simple.py
[2023-06-24T10:50:09.684+0000] {spark_submit.py:492} INFO - WARNING: An illegal reflective access operation has occurred
[2023-06-24T10:50:09.684+0000] {spark_submit.py:492} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/opt/spark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2023-06-24T10:50:09.684+0000] {spark_submit.py:492} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2023-06-24T10:50:09.684+0000] {spark_submit.py:492} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2023-06-24T10:50:09.684+0000] {spark_submit.py:492} INFO - WARNING: All illegal access operations will be denied in a future release
[2023-06-24T10:50:09.728+0000] {spark_submit.py:492} INFO - Exception in thread "main" org.apache.spark.SparkException: Master must either be yarn or start with spark, mesos, k8s, or local
[2023-06-24T10:50:09.728+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.error(SparkSubmit.scala:972)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.prepareSubmitEnvironment(SparkSubmit.scala:238)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:898)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:180)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:203)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:90)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1043)
[2023-06-24T10:50:09.729+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1052)
[2023-06-24T10:50:09.730+0000] {spark_submit.py:492} INFO - at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
[2023-06-24T10:50:09.760+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 423, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master  --name arrow-spark /opt/***/dags/spark_etl_simple.py. Error code is: 1.
[2023-06-24T10:50:09.764+0000] {taskinstance.py:1321} INFO - Marking task as FAILED. dag_id=spark_***_dag_v2, task_id=spark_submit_task, execution_date=20230624T105008, start_date=20230624T105008, end_date=20230624T105009
[2023-06-24T10:50:09.779+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 61 for task spark_submit_task (Cannot execute: spark-submit --master  --name arrow-spark /opt/***/dags/spark_etl_simple.py. Error code is: 1.; 1732)
[2023-06-24T10:50:09.820+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-06-24T10:50:09.834+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
